Formula/watchman.rb

```diff
@@ -6,21 +6,13 @@ class Watchman < Formula
   head "https://github.com/facebook/watchman.git"
 
   stable do
-    url "https://github.com/facebook/watchman/archive/v4.9.0.tar.gz"
```

Comment:
The whole stable block should be removed, same as the `revision` at the top

Reply:
Done! 👍